pennylane/math/utils.py

```diff
@@ -404,6 +404,8 @@ def function(x):
         import jax
         from jax.interpreters.partial_eval import DynamicJaxprTracer
 
+        from pennylane.capture import TransformTracer

```

Comment:
Wanted to avoid importing top-level since this file does not import pennylane at all.

Reply:
What happens if we just check `isinstance(tensor, jax.core.Tracer)` instead of the more specific `DyanmicJaxprTracer`?

Reply:
I'll try it out.

Reply:
Works. Resolving.